packages/fxa-settings/src/components/Settings/PageSettings/index.tsx

```diff
@@ -72,7 +79,7 @@ export const PageSettings = (_: RouteComponentProps) => {
     route: '/settings/account_recovery',
     dismissKey: 'account-recovery-dismissed',
     metricsKey: 'create_recovery_key',
-    isVisible: !recoveryKey.exists,
+    isVisible: estimatedSyncDeviceCount > 0 && !recoveryKey.exists,
```

Comment:
The ticket also mentions increasing the space on desktop between "Create" and the "close" button but this change looks good, thanks for the quick patch!

Reply:
@LZoog Updated this, figma on top and this PR on bottom. I'll probably land this so we can get it out on point release.

<img width="294" alt="Screenshot 2024-09-17 at 1 08 50 PM" src="https://github.com/user-attachments/assets/e556c650-9031-449d-86ba-9a27c7f0e229">

Reply:
FYI, Eduardo had asked to align the Banner button with the Row CTA buttons below - that's why it ended up so close to the close button. Only alternative I could see was adding more vertical padding, but that didn't look great either...